java/vector/src/main/java/org/apache/arrow/vector/UInt2Vector.java

```diff
@@ -62,7 +64,19 @@ public MinorType getMinorType() {
    |          vector value retrieval methods                        |
    |                                                                |
    *----------------------------------------------------------------*/
-
+  /**
+   * Given a data buffer, get the value stored at a particular position
+   * in the vector.
+   *
+   * <p>This method is mainly meant for integration tests.
+   *
+   * @param buffer data buffer
+   * @param index position of the element.
+   * @return value stored at the index.
+   */
+  public static long get(final ArrowBuf buffer, final int index) {
```

Comment:
any reason to not return char and returning a long..

Reply:
Copy paste bug will fix